tests/test_postgres.py

```diff
@@ -19,6 +19,7 @@ class TestTwoDatabases(test.SimpleTestCase):
         if Tortoise._inited:
             await Tortoise._drop_databases()
 
+    @test.skip("somehow doesn't work with pools?")
     async def test_schema(self):
```

Comment:
I don't see anything in this test that should break with pools... Curious.

Reply:
somehow when using pools it just takes accepts the invalid scheme and runs with it, no error raised at all

Reply:
:thinking: Is the scheme somehow swallowed? I know this needs to work as it is actually used by users. The last time I broke this I got two bugreports within hours of the broken release...

Reply:
i'll see if i can make it work, maybe it'll realize the issue if i actually interact with the database?